packages/ckeditor5-ui/src/toolbar/balloon/balloontoolbar.ts

```diff
@@ -275,6 +280,39 @@ export default class BalloonToolbar extends Plugin {
 		}
 	}
 
+	/**
+	 * Add or remove editable elements to the focus tracker. It watches added and removed roots
+	 * and adds or removes their editable elements to the focus tracker.
+	 */
+	private _trackEditablesFocusableElements() {
+		const { editor, focusTracker } = this;
+		const { ui } = editor;
+
+		// Add existing editable elements to the focus tracker.
+		for ( const editableName of ui.getEditableElementsNames() ) {
+			const editableElement = ui.getEditableElement( editableName );
+
+			focusTracker.add( editableElement! );
+		}
+
+		// Track added / updated editable elements.
+		ui.on<EditorUISetEditableElementEvent>( 'setEditableElement', ( evt, args ) => {
+			const [ , domElement ] = args;
+
+			focusTracker.add( domElement );
+		}, { priority: 'low' } );
```

Comment:
Just a suggestion, maybe this way would be more clear.

```suggestion
		ui.on<EditorUISetEditableElementEvent>( 'setEditableElement', ( evt, [ , domElement ] ) => {
			focusTracker.add( domElement );
		}, { priority: 'low' } );
```